File: Programmers/Python/Lv.0/[Lv.0] 옷가게 할인 받기/[Lv.0] 옷가게 할인 받기_01.py
def solution(price):
    # 할인가 = price * 할인율(100 - 할인%)
    discount = { 500000: 0.8, 300000: 0.9, 100000: 0.95, 0: 1 }

    for i, j in discount.items():
        if price >= i:
            return int(price * j)

File: Programmers/코딩테스트 입문/Java/Lv.0/[Lv.0] 옷가게 할인 받기/[Lv.0] 옷가게 할인 받기_01.java
class Solution {
    public int solution(int price) {
        // 할인가 = price * 할인율(100 - 퍼센트(%))
        return (int) (price >= 500_000 ? price - price * 0.2 : price >= 300_000 ? price - price * 0.1 : price >= 100_000 ? price - price * 0.05 : price);
    }
}
